verification/curator-service/ui/src/constants/types.ts

```diff
@@ -36,9 +36,30 @@ export const MapLink: IOutbreakEnvironments = {
         local: 'http://dev-map.covid-19.global.health/',
         locale2e: 'http://dev-map.covid-19.global.health/',
         dev: 'http://dev-map.covid-19.global.health/',
-        qa: 'http://qa-map.covid-19.global.health',
+        qa: 'http://dev-map.covid-19.global.health',
         prod: 'https://map.covid-19.global.health/',
     },
+    Marburg: {
```

Comment:
Could this use environment variables as well? Particularly if we want partners to use this, easier to ask them to update env vars, otherwise we would have create MapLinks in our codebase, or they would have to fork our code to add their own MapLinks.

Reply:
That makes sense I'll update that